src/components/ComputeModal.js

```diff
@@ -23,7 +23,7 @@ import { betaVersionTag } from 'src/libs/logos'
 import * as Nav from 'src/libs/nav'
 import { useOnMount } from 'src/libs/react-utils'
 import {
-  computeStyles, defaultComputeRegion, defaultComputeZone, defaultDataprocMachineType, defaultDataprocMasterDiskSize, defaultDataprocWorkerDiskSize,
+  computeStyles, defaultAutoPause, defaultAutoPauseThreshold, defaultComputeRegion, defaultComputeZone, defaultDataprocMachineType, defaultDataprocMasterDiskSize, defaultDataprocWorkerDiskSize,
```

Comment:
Consider reducing the number of explicit imports here, which is pretty noisy:
1. `import * as ru from 'src/libs/runtime-utils'` and use as `ru.computeStyles` etc.
2. Create a `defaults` object in runtime-utils and use that, i.e.,
```javascript
const [computeConfig, setComputeConfig] = useState(_.merge(ru.defaultComputeConfig, {/* overrides */}))
```